scripts/pylarion-cmd

```diff
@@ -1,344 +1,237 @@
 #!/usr/bin/env python
 
 import os
-import sys
-import commands
-import getopt
-import ConfigParser
-import shutil
-from pylarion.cli.CliDoc import CliDoc
-from pylarion.cli.CliWorkItem import CliWorkItem
-from pylarion.cli.CliTestRun import CliTestRun
+from pylarion.cli.cmd import Repo
+from pylarion.cli.cmd import Config
+from pylarion.cli.cmd import CmdList
+from pylarion.cli.cmd import CmdUpdate
```

Comment:
Noticed you have merge all files under src/pylarion/cli/* into one cmd.py.
Two suggestions for your consideration:
 - to avoid an extra confusing layer, we can simply put the src/pylarion/cli/cmd.py as src/pylarion/cmd.py
 - then "import pylarion.cmd" here, and use cmd.*() when calling those functions in the rest of pylarion-cmd.py to avoid potential namespace polution.